### PR TITLE
docs: fix cwd description for shell.writeShortcutLink

### PR DESCRIPTION
**Pull request by miniak**

### CODE REVIEW

File: docs/api/web-contents.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ Emitted when the cursor's type changes. The `type` parameter can be `default`,
 `not-allowed`, `zoom-in`, `zoom-out`, `grab`, `grabbing`, `custom`.
 
 If the `type` parameter is `custom`, the `image` parameter will hold the custom
-cursor image in a `NativeImage`, and `scale`, `size` and `hotspot` will hold
```
Comment (MarshallOfSound) [inline docs/api/web-contents.md]: I want to call scope creep on this formatting thing.  But I'm also not sure if anyone would care :/

Comment (miniak) [inline docs/api/web-contents.md]: I have trailing whitespace stripping enabled in my editor. I can revert this line.

**Comment by kevinsawicki** [comment]
Thanks 👍
